src/views/AcademicCheckIn/components/EmergencyContactUpdate/index.js

```diff
@@ -0,0 +1,220 @@
+// import React, { useState, useEffect } from 'react';
+import {
+  FormControl,
+  FormControlLabel,
+  InputLabel,
+  Checkbox,
+  Input,
+  Grid,
+  Typography,
+  Box,
+} from '@material-ui/core';
+import { gordonColors } from 'theme';
+import { phoneMaskUS, phoneMaskINTL } from 'views/AcademicCheckIn/components/UpdatePhone/';
+
+const EmergencyContactUpdate = ({
+  emergencyContact1,
+  emergencyContact2,
+  emergencyContact3,
+  emergencyContactINTL1,
+  emergencyContactINTL2,
+  emergencyContactINTL3,
+  handleChangeEmergContact1,
+  handleChangeEmergContact2,
+  handleChangeEmergContact3,
+  handleCheckEmergContact1,
+  handleCheckEmergContact2,
+  handleCheckEmergContact3,
+}) => {
+  let cyan = gordonColors.primary.cyan;
+
+  return (
+    <Grid container justifyContent="center" alignItems="center" direction="column">
+      <Typography align="center" variant="h5" gutterBottom style={{ color: cyan }}>
+        Step 1: Enter your Emergency Contact Information
+      </Typography>
+      <Typography variant="h6" gutterBottom>
+        <b>Who are your emergency contacts?</b>
+      </Typography>
+      <Typography variant="body1" gutterBottom>
+        HIPAA regulations prohibit sharing of medical information regarding anyone age 18 or over
+        without consent.
+      </Typography>
+      <Typography variant="body1" gutterBottom>
+        By listing emergency contacts below, I authorize the Gordon College staff to share medical
+        information with my emergency contact(s) during my enrollment at Gordon College. I hereby
+        authorize Gordon College to contact the following person(s) listed below in the event of a
+        medical emergency.
+      </Typography>
+      {createEmergencyContactFields(
+        emergencyContact1,
+        emergencyContactINTL1,
+        handleChangeEmergContact1,
+        handleCheckEmergContact1,
+      )}
+      {createEmergencyContactFields(
+        emergencyContact2,
+        emergencyContactINTL2,
+        handleChangeEmergContact2,
+        handleCheckEmergContact2,
+      )}
+      {createEmergencyContactFields(
+        emergencyContact3,
+        emergencyContactINTL3,
+        handleChangeEmergContact3,
+        handleCheckEmergContact3,
+      )}
+      <Grid item>
+        <br />
+      </Grid>
+      <Typography variant="body2" gutterBottom>
+        If, during the semester your emergency contact information must be updated, please contact
+        Student Life at 978.867.4263.
+      </Typography>
+      <Typography variant="body2" gutterBottom>
+        For international phone numbers, check the "Non-US Number" checkbox to enter information.
```

Comment:
These instructions need to be updated. The checkbox is labelled `International number` now.
```suggestion
        For international phone numbers, check the "International #" checkbox.
```